src/main/java/com/mraof/minestuck/entity/consort/ConsortEntity.java

```diff
@@ -62,15 +65,16 @@ public ConsortEntity(EnumConsort consortType, EntityType<? extends ConsortEntity
 	
 	public static AttributeModifierMap.MutableAttribute consortAttributes()
 	{
-		return MobEntity.createMobAttributes().add(Attributes.MAX_HEALTH, 10).add(Attributes.MOVEMENT_SPEED, 0.25);
+		return MobEntity.createMobAttributes().add(Attributes.MAX_HEALTH, 10).add(Attributes.MOVEMENT_SPEED, 0.35);
```

Comment:
This movement speed seems to match the animation better, was that why it was changed?

Reply:
honestly its just an arbitrary value that felt good when i tested